eng/pipelines/common/templates/pipeline-with-resources.yml

```
@@ -60,6 +60,9 @@ resources:
     - container: Browser_wasm
       image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-webassembly
 
+    - container: wasi_wasm
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-webassembly-staging
```

we can move this to `ubuntu-20.04-webassembly` now
```suggestion
      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-webassembly
```